src/pages/item/_.vue

```diff
@@ -125,7 +124,7 @@
 </template>
 
 <script>
-  import isEmpty from 'lodash/isEmpty';
+  import isEmpty from 'lodash/isEmpty.js';
```

Comment:
What does this achieve?

Reply:
At some point test coverage failed locally and I was under the assumption this fixed it. It does not. I see now the svg transformer that I had added to the jest config is the problem. Above change is reverted and transformer fixed.